CONTAINERS.md

```diff
@@ -35,17 +35,33 @@ It also runs database migrations.
 We also provide a base image which can be reused for Hyrax applications: `hyrax`.
 
 ```sh
-echo "FROM hyrax" > Dockerfile
+echo "FROM samveralabs/hyrax" > Dockerfile
 ```
 
 ### Maintaining
 
-At the moment, we don't publish hyrax images anywhere (TODO). If you want to use them, you need to build them yourself:
+We publish several Hyrax images to hub.docker.com under the
+[`samveralabs` group][dockerhub-samveralabs]. To build them, do:
 
 ```sh
 # build an image for an app using Postgresql (`gem 'pg'`)
-docker build --target hyrax --tag hyrax .
+docker build --target hyrax --tag samveralabs/hyrax:(git rev-parse HEAD) .
+docker push samveralabs/hyrax:(git rev-parse HEAD)
```

Comment:
I'm assuming the `docker push` command would push up the image to `samveralabs`.  Is that correct?  Should we document that? What about how to get permission to do that?

Reply:
that's right. this is to publish the image.

right now, there's no strategy for handing out permissions. i had to bang the drum to get them myself.